src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt

```diff
@@ -0,0 +1,152 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+internal class SentenceCalculationTest {
+
+  @Test
+  fun calculatedReleaseDate() {
```

Comment:
minor one, but in kotlin you can name test functions like this in cases where we need to be more descriptive
fun `Calculate a release date`() {